backend/src/modules/Announcements/service/Announcements.service.js

```diff
@@ -15,4 +15,10 @@ export class AnnouncementsService {
     // TODO: Zwrócić listę announcementów tego boarda po dodaniu zamiast tego niżej
     return [createdAnnouncement];
   }
+
+  async getAnnouncement(announcementId) {
```

Comment:
Moglibyśmy tutaj bardziej biznesowo nazwać w serwisie.
Jakbym miał te ogłoszenia, to powiedziałbyś do mnie "get announcment" czy raczej np.  "find announcement", a może "find one announcement"?
Warto, żebyśmy coraz bardziej w tym kierunku zaczynali myśleć. :)

```suggestion
  async findAnnouncement(announcementId) {
```

Reply:
done